db/migrate/20150106161639_cleanup_endorsements.rb

```diff
@@ -0,0 +1,11 @@
+class CleanupEndorsements < ActiveRecord::Migration
+  def up
+    Endorsement.find_each(batch_size: 100) do |endo|
```

Comment:
I thought finding in batches wouldn't clog the memory.. Not sure about `delete_all` way of handling though. You want me to rewrite it?